src/main/scala/uniq/mqttchat/client/AdminClient.scala

```diff
@@ -0,0 +1,216 @@
+
+// copyright
+
+package uniq.mqttchat.client
+
+// scalastyle:off
```

Comment:
Merger all import with underscore `_` within `// scalastyle:off underscore.import` and `// scalastyle:on ...`